source/PlayState.hx

```diff
@@ -3908,6 +3908,20 @@ class PlayState extends MusicBeatState
 		}
 	}
 	#end
+	
+	public var screenGoesBlack:Bool = true;
+	function blackScreen(?screenGoesBlack:Bool = true) { // this can also be used for winter horrorland
+		if (screenGoesBlack) {
+			camGame.visible = false; //Setting both cams go invisible makes it go black? idk why lol
+			camHUD.visible = false;
+			camOther.visible = false; //why not?
+		} else {
+			camGame.visible = true; //Setting them back
+			camHUD.visible = true;
+			camOther.visible = true;
+		}
+		setOnLuas('blackScreen', screenGoesBlack)
```

Comment:
A bit of a nitpick, but you could also just make the function parameter `blackScreen` as well for consistency.